com/company/PrintAndCats/Main.java

```diff
@@ -12,10 +12,10 @@ public static void main(String[] args) {
 
```

Comment:
1. Create two cats initialized by constructor with arguments (name, age)
2. Make name and age private fields
3. Make getter/setter for name and age
4. Override toString Method to print name, age

Reply:
Ok